index.html

```diff
@@ -1022,6 +1022,12 @@ <h2>DID Parameters</h2>
 <a href="#did-parameters">DID Parameters</a> is OPTIONAL.
         </p>
 
+        <p class="note">
+It's generally expected that DID URL dereferencer implementations will
```

Comment:
```suggestion
It is generally expected that DID URL dereferencer implementations will
```

Reply:
I always forgot about the contraction usage rule. Thanks.